resources/metadata/detect.yaml

```diff
@@ -85,6 +85,27 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+      - name: dependencyPath
+        description: Absolute Path of the dependency management file of the project. This path represents the folder which contains the pom file, package.json etc. If the project contains multiple pom files, provide the path to the parent pom file or the base folder of the project
+        aliases:
+          - name: detect/dependencyPath
+        type: "string"
+        default:
+          - "."
```

Comment:
⚠️  The type here is `string` but the default is a `list`.

Reply:
Thank you.
Fixed in commit https://github.com/SAP/jenkins-library/pull/2508/commits/08b3e22acfb374c74852e456dd8051dff3fb8b8d